ast/compile.go

```diff
@@ -5406,3 +5414,5 @@ func (rs *refSet) Sorted() []*Term {
 	})
 	return terms
 }
+
+var refBuiltins = map[string]struct{}{ObjectGet.Name: {}, Member.Name: {}, MemberWithKey.Name: {}}
```

Comment:
🔍 We need a good way to declare builtins as "lazy ref"-able. We cannot pull this off without changing the code of the builtins, so it needs to be opt-in.